src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/controllers/PostSubmitCallbackController.java

```diff
@@ -18,12 +15,12 @@
 import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.Callback;
 import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.PostSubmitCallbackResponse;
 import uk.gov.hmcts.reform.iacaseapi.infrastructure.PostSubmitCallbackDispatcher;
-import uk.gov.hmcts.reform.iacaseapi.infrastructure.serialization.Deserializer;
+
 
 @Api(
```

Comment:
Is there a way to document the authorization headers required?
I see `@Api` has a `authorizations` property thing but I don't know how that works?

Reply:
I think you can although I'll have to find out exactly how.